recipe/meta.yaml

```diff
@@ -38,13 +38,13 @@ outputs:
       build:
         - {{ compiler('c') }}
       host:
-        - zlib >=1.2.3
-        - openssl >=1.0.1  # [unix]
+        - zlib
+        - openssl  # [unix]
         - libev >=4.11
         - c-ares >=1.7.5
       run:
-        - zlib >=1.2.3
-        - openssl >=1.0.1  # [unix]
+        - zlib
+        - openssl  # [unix]
         - libev >=4.11
         - c-ares >=1.7.5
         - libstdcxx-ng  # [linux]
```

Comment:
isn't this added from the compiler stuff on linux anyways?

Reply:
turns out its not?